### PR TITLE
Develop to master

### PR DESCRIPTION
**Pull request by ThrawnCA**
- Merge theme into this extension to simplify deployments
- Split Pylons routing from Flask
- Split Paster CLI from Click
- Define Webassets config for static resources

### CODE REVIEW

File: .docker/test.ini [hunk 1]
```diff
@@ -94,7 +97,7 @@ ckan.redis.url = redis://redis:6379
 #       Add ``datapusher`` to enable DataPusher
 #       Add ``resource_proxy`` to enable resource proxying and get around the
 #       same origin policy
-ckan.plugins = stats text_view image_view recline_view datastore data_qld_theme datarequests data_qld_resources data_qld_integration scheming_datasets validation dcat qa archiver report qgovext data_qld_reporting ytp_comments
+ckan.plugins = stats text_view image_view recline_view datastore data_qld datarequests scheming_datasets validation dcat qa archiver report qgovext ytp_comments
```
Comment (duttonw) [inline .docker/test.ini:100]: did we update the order on opsworks cookbook?

File: ckanext/data_qld/templates/robots.txt [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+Disallow: /sq/
+Disallow: /sr/
+Disallow: /sr_Latn/
+Disallow: /sv/
+Disallow: /tr/
+Disallow: /uk_UA/
+Disallow: /zh_CN/
+Disallow: /zh_TW/
+# presigned S3 URLs that are only temporary
+Disallow: /ckan-*-attachments-*/
```
Comment (duttonw) [inline ckanext/data_qld/templates/robots.txt:55]: Is this still a problem now?

**Review by duttonw — APPROVED** [review]

File: .docker/test.ini (hunk 1 above)
Comment (ThrawnCA) [reply .docker/test.ini:100]: This is just for CI testing. OpsWorks will still generate the individual plugin names on actual deployments (but all except `data_qld_integration` will point to an empty placeholder plugin).

File: ckanext/data_qld/templates/robots.txt (hunk 2 above)
Comment (ThrawnCA) [reply ckanext/data_qld/templates/robots.txt:55]: I don't think we've updated every legacy resource, so I wouldn't change this yet.

**Pull request merged**